<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_feed</class>
 <widget class="QDialog" name="main_feed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="create_post">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Post</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="post_title">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QPushButton" name="messages">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>69</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Messages</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="feed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Feed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_out">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>@humanefisher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_hashtag">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Hashtags</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profile">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>70</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QLabel" name="post_timestamp">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>200</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>50</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="post_profilePicture">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../database/pictures/4f02e903d4e76b458f3e663405a51679.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fullName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PlaceHolder</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>150</y>
     <width>20</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="report">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../database/pictures/1045119-200.png</normaloff>../../database/pictures/1045119-200.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>20</width>
     <height>311</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="post_content">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>581</width>
     <height>131</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No posts detected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_post">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_post">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>470</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="post_profilePicture_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../database/pictures/4f02e903d4e76b458f3e663405a51679.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="post_fullName">
      <property name="text">
       <string>Full Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="post_username">
      <property name="text">
       <string>@username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="post_hashtag">
      <property name="text">
       <string>Hashtags</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="go_to_user">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Go to User</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
